src/platform/haswell/include/arch/xtensa/config/core-isa-bdw.h

```diff
@@ -89,7 +89,7 @@
 #define XCHAL_HAVE_PSO			0	/* Power Shut-Off */
 #define XCHAL_HAVE_PSO_CDM		0	/* core/debug/mem pwr domains */
 #define XCHAL_HAVE_PSO_FULL_RETENTION	0	/* all regs preserved on PSO */
-#define XCHAL_HAVE_THREADPTR		1	/* THREADPTR register */
+#define XCHAL_HAVE_THREADPTR		0	/* THREADPTR register */
```

Comment:
I think we should update the whole header.
There is a high probability, that the rest of the values is also wrong.

Reply:
I checked the others and they looked correct.